draft-irtf-cfrg-hpke.md

```diff
@@ -554,6 +554,38 @@ def Context.Open(aad, ct):
   return pt
 ~~~~~
 
+# Single-Shot APIs
+
+In many cases, applications encrypt only a single message to a recipient's public key. 
+This section provides templates for HPKE APIs that implement "single-shot" encryption 
+and decryption using APIs specified in {{hpke-kem}} and {{hpke-dem}}:
+
+~~~~~
+def Seal<MODE>(pkR, info, aad, pt, ...):
+  enc, ctx = SetupI<MODE>(pkR, info, ...)
```

Comment:
We should make sure that the templates actually expand here.  Based on the AuthPSK example above, it looks like there are at least two issues:

* `SetupR<MODE>` vs. `Setup<MODE>R`
* Parameter order (standard parameters first)

Reply:
Great catch! Should be fixed now.